src/modules/font.c

```diff
@@ -24,76 +16,35 @@ void ffPrintFont(FFinstance* instance)
         return;
     }
 
-    FFfont plasma;
-    ffFontInitQt(&plasma, plasmaRaw->chars);
-
-    FFfont gtk2;
-    ffFontInitPango(&gtk2, gtk2Raw->chars);
-
-    FFfont gtk3;
-    ffFontInitPango(&gtk3, gtk3Raw->chars);
-
-    FFfont gtk4;
-    ffFontInitPango(&gtk4, gtk4Raw->chars);
-
-    FFstrbuf gtk;
-    ffStrbufInitA(&gtk, 64);
-    ffParseGTK(&gtk, &gtk2.pretty, &gtk3.pretty, &gtk4.pretty);
-
-    if(instance->config.font.outputFormat.length == 0)
+    for(uint32_t i = 0; i < list.length; ++i)
     {
-        ffPrintLogoAndKey(instance, FF_FONT_MODULE_NAME, 0, &instance->config.font.key);
-        if(plasma.pretty.length > 0)
+        FFFontResult* font = (FFFontResult*)ffListGet(&list, i);
+        if(instance->config.font.outputFormat.length == 0)
         {
-            ffStrbufWriteTo(&plasma.pretty, stdout);
-            fputs(" [QT]", stdout);
-
-            if(gtk.length > 0)
-                fputs(", ", stdout);
+            FFstrbuf key;
+            ffStrbufInitF(&key, "%s (%s)", FF_FONT_MODULE_NAME, font->type);
+            ffPrintLogoAndKey(instance, key.chars, 0, &instance->config.font.key);
+            puts(font->fontPretty.chars);
+            ffStrbufDestroy(&key);
         }
-        ffStrbufPutTo(&gtk, stdout);
-    }
-    else
-    {
-        ffPrintFormat(instance, FF_FONT_MODULE_NAME, 0, &instance->config.font, FF_FONT_NUM_FORMAT_ARGS, (FFformatarg[]){
-            {FF_FORMAT_ARG_TYPE_STRBUF, plasmaRaw},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &plasma.name},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &plasma.size},
-            {FF_FORMAT_ARG_TYPE_LIST,   &plasma.styles},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &plasma.pretty},
-            {FF_FORMAT_ARG_TYPE_STRBUF, gtk2Raw},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk2.name},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk2.size},
-            {FF_FORMAT_ARG_TYPE_LIST,   &gtk2.styles},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk2.pretty},
-            {FF_FORMAT_ARG_TYPE_STRBUF, gtk3Raw},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk3.name},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk3.size},
-            {FF_FORMAT_ARG_TYPE_LIST,   &gtk3.styles},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk3.pretty},
-            {FF_FORMAT_ARG_TYPE_STRBUF, gtk4Raw},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk4.name},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk4.size},
-            {FF_FORMAT_ARG_TYPE_LIST,   &gtk4.styles},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk4.pretty},
-            {FF_FORMAT_ARG_TYPE_STRBUF, &gtk}
-        });
+        else
+        {
+            ffPrintFormat(instance, FF_FONT_MODULE_NAME, 0, &instance->config.font, FF_FONT_NUM_FORMAT_ARGS, (FFformatarg[]){
+                {FF_FORMAT_ARG_TYPE_STRING, font->type},
+                {FF_FORMAT_ARG_TYPE_STRBUF, &font->fontPretty}
+            });
+        }
+        ffStrbufDestroy(&font->fontPretty);
```

Comment:
I don't like the idea of printing QT and GTK in separate lines. Can we keep them inline?